### ISSUE TITLE
Method to enable font ligatures in the 2.0.0 series

### ISSUE THREAD

**Issue by mapitman** [zeit/hyper#2793]
- [ ] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: Antergos Linux
- **Hyper.app version**: 2.0.0-canary.15
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: https://gist.github.com/mapitman/dd182521e20332319049985fcdc1c6b5
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: N/A
- **The issue is reproducible in vanilla Hyper.app**: 
## Issue

All of the methods I've found for enabling font ligatures do not work in the 2.0.0 series of releases. Is there an example of the correct CSS for 2.0.0?

**Comment by chaseleinart** [comment]
@mapitman - Hi Mark - according to comments on #914, hyper does not support ligatures due to current limitations with xterm3. That project still has ligatures as an open feature request according to the link below

- https://github.com/xtermjs/xterm.js/issues/958

**Comment by mapitman** [comment]
Thanks @chaseleinart . I'll go ahead and close this issue then.

**Comment by SirWindfield** [comment]
Looks like they published an official addon to enable font ligatures support: https://github.com/xtermjs/xterm-addon-ligatures

**Comment by BenMcH** [comment]
@SirWindfield Great catch! I just built hyper locally with that addon. I had to precompile the library (because it's typescript) and I ended up throwing it in the lib directory, but I was able to get ligatures working.

If I can figure out a better way to include this change and get it building, I'll submit a PR!

@mapitman We may want to reopen this now that it's been shown to work with the addon from xterm.
